### PR TITLE
control-service: introduce data job deployment entity

### PR DESCRIPTION
**Pull request by mivanov1988**
# Why
As part of the VEP-2272, we need to switch the source of truth from Kubernetes to the database.

# What
Implemented a database table named "data_job_deployment," designated for the storage of all data associated with data job deployments.

# Testing Done
Unit tests

Signed-off-by: Miroslav Ivanov miroslavi@vmware.com

### CODE REVIEW

**Review by antoniivanov — APPROVED** [review]

File: projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  private String pythonVersion;
+
+  private String gitCommitSha;
+
+  private Float resourcesCpuRequest;
+
+  private Float resourcesCpuLimit;
+
+  private Integer resourcesMemoryRequest;
```
Comment (dakodakov) [inline projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java:41]: What are the units of measurement for CPU and Memory? Please include them in the name of the variables.

**Review by dakodakov — APPROVED** [review]

File: projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java (hunk 1 above)
Comment (mivanov1988) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java:41]: @dakodakov For memory will be resourcesMemoryRequestMi. What is your suggestion for the CPU?

Comment (dakodakov) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java:41]: ....CPULimitCores? I don't have a strong opinion.

Comment (mivanov1988) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java:41]: Thanks, I will make the changes as part of the next PR.

**Pull request merged**